hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/sender/http/JettyMessageSenderTest.java

```diff
@@ -152,7 +163,14 @@ public void shouldSendRetryCounterInHeader() {
     @Test
     public void shouldSendAuthorizationHeaderIfAuthorizationProviderAttached() {
         // given
-        JettyMessageSender messageSender = new JettyMessageSender(client, address, Optional.of(m -> "Basic Hello!"), 1000, new DefaultHttpMetadataAppender());
+        HashMap<String,String> authData = new HashMap<String,String>();
```

Comment:
but it was already tested - provider returned "Hello world" and that made it to the header